x/docs/track.rb

```diff
@@ -0,0 +1,54 @@
+module X
+  module Docs
+    class Track
+      attr_reader :track
+      def initialize(track)
```

Comment:
following my previous question, if the answer is nowhere but inside this class, then it can be simplified and be more on compliance with Demeter low when converting to a `SimpleDelegator` class:

```ruby
module X
  module Docs
    class Track < SimpleDelegator
      # ...
      def substitutions
        {
          'LANGUAGE' => language,
          'TRACK_ID' => id,
          'REPO' => repository,
          'CHECKLIST_ISSUE' => checklist_issue,
        }
      end
    end
  end
end
```